src/NUnitEngine/nunit.engine/Internal/ResultHelper.cs

```diff
@@ -112,8 +118,9 @@ public static void InsertEnvironmentElement(this XmlNode resultNode)
             env.AddAttribute("machine-name", Environment.MachineName);
             env.AddAttribute("user", Environment.UserName);
             env.AddAttribute("user-domain", Environment.UserDomainName);
-            env.AddAttribute("culture", System.Globalization.CultureInfo.CurrentCulture.ToString());
-            env.AddAttribute("uiculture", System.Globalization.CultureInfo.CurrentUICulture.ToString());
+            env.AddAttribute("culture", CultureInfo.CurrentCulture.ToString());
+            env.AddAttribute("uiculture", CultureInfo.CurrentUICulture.ToString());
+            env.AddAttribute("os-architecture", Environment.GetEnvironmentVariable("PROCESSOR_ARCHITECTURE"));
```

Comment:
Do we know that this environment variable is set on all platforms? Does it work on Mac, Linux, Android (under Xamarin), etc?